web_console/dynamic-plugin/deploy-plugin-cluster.adoc

```diff
@@ -12,11 +12,24 @@ include::modules/build-image-docker.adoc[leveloffset=+1]
 
 include::modules/deployment-plug-in-cluster.adoc[leveloffset=+1]
 
+include::modules/dynamic-plugin-proxy-service.adoc[leveloffset=+1]
+
+ifndef::openshift-rosa,openshift-dedicated[]
+[role="_additional-resources"]
+[id="proxy-service-additional-resources"]
+== Additional resources
```

Comment:
This looks like a module-level add'l resources list rather than assembly-level

```suggestion
[role="_additional-resources"]
.Additional resources
```